src/pages/index.html

```diff
@@ -85,7 +148,55 @@ <h1>totem dov</h1>
         <span class="badge">loremmm</span>
         <span class="badge badge--primary">loremmm</span>
       </div>
-      <div class="col-3">c</div>
+      <div class="col-3">
+        <div class="card--rounded">
+          <div class="card--rounded__box-img">
```

Comment:
Image is not specific to the rounded card, but to *card* in general.

`card--rounded` is just a modifier for `card`. Instead, use `card__image-containter`, for example.